README.md

```diff
@@ -1,21 +1,93 @@
+
+While in the `pudl-zenodo-storage` environment, navigate to the `pudl` repo and run:
 ```
-conda env create -f environment.yml
-conda activate pudl-zenodo-storage
+$ pip install -e ./
 ```
 
-## zs
+You don't need to worry about the `fields.py` module until you're ready to transform the
+data in pudl.
+
+Now, come back to this repo and create a module for the dataset in the `frictionless`
+directory. Give it the same name as the key you made for the data in the SOURCE dict.
+Use the existing modules as a model for your new one. The main function is called
+`datapackager()` and it serves to produce a json for the Zenodo archival collection.
+
+Lastly, you need to:
+- Create archive metadata for the new dataset in the `zs/metadata.py` module.
+- Add the chosen deposition name to this list of acceptable names output with the `zenodo_store --help` flag. See `parse_main()` in `zs.cli.py`.
+- Add specifications for your new deposition in the `archive_selection()` function also in `zs.cli.py`.
+
+## Updating an Existing Data Source
+If updating an existing data source--say, one that as released a new year's worth of
+data--you don't need to add any new metadata to the `pudl` repo. Simply run the scraper
+for the data and then run the Zenodo script as described below. The code was built to
+detect any changes in the data and automatically create a new version of the same deposition when uploaded.
+
+## Running the Zenodo Archiver Script
+Before you can archive data, you'll need to run the scrapy script you just created in the `pudl-scrapers` repo. Once you've scraped the data, then you can come back and run the archiver. This script, `zenodo_store` gets defined as an entry point in `setup.py`.
+
+Next, you'll need to define `ZENODO_SANDBOX_TOKEN_UPLOAD` and `ZENODO_TOKEN_UPLOAD` environment variables on your local machine. You'll need to ask us for these!
+
+The `zenodo_store` script requires you to include the name of the Zenodo deposition as an argument. This is a string value that indicates which dataset you're going to upload. Use the `--help` flag to see a list of supported strings. You can also find a list of the deposition names in the `archive_selection()` function in the `cli.py` module.
+
+When you're testing an archive, you'll want to make sure you use the Zenodo
+sandbox rather than the official Zenodo archive (see above for more info about the sandbox). Adding the `--verbose` flag will print out logging messages that are helpful for debugging. Adding the `--noop` flag will show you whether your the data you scraped
+is any different from the data you already have uploaded to Zenodo without uploading anything (so long as there is an existing upload to compare it to).
+
+If the dataset is brand new, you'll also need to add the `--initialize` flag so that it knows to create a new deposition for the data.
+
+```
+$ zenodo_store newdata --noop
+$ zenodo_store newdata --sandbox --verbose --initialize
```

Comment:
It's nice to provide the output from example commands in addition to the commands themselves so folks know what to expect / can see if there's something acting strangely.